charts/sonarqube/values.yaml

```diff
@@ -86,6 +86,15 @@ ingress:
   #   hosts:
   #     - chart-example.local
 
+# Create Route for OpenShift deployments. Set enabled=true if you want Route to be automatically generated based on service and namespace names.
+route:
```

Comment:
I think the new values should at least be mentioned in the `README.md`. don't you think?

Reply:
I added config params to README, although I think that most users will read the descriptions from the `values.yaml` file anyway.

Reply:
I agree with you both. README.md is the main file anyone will read.